builder/js/ncf.js

```diff
@@ -1209,8 +1209,7 @@ $scope.onImportFileChange = function (fileEl) {
 
   // Delete a technique
   $scope.deleteTechnique = function() {
-    var data = {params: {path: $scope.path}};
-    $http.delete("/ncf/api/techniques/"+$scope.selectedTechnique.bundle_name, data).
+    $http.delete("/ncf/api/techniques/"+$scope.selectedTechnique.bundle_name).
```

Comment:
shouldn't this be ported to rudder API too?

Reply:
Totally ! when i passed on this I thought i would do it in another PR, and it will give us a lot more possibilities, like checking if there is directives applied to this technique, and prevent deletion of the technique, or provide an option to force deletion and also delete Directives and Techniques ...,

Reply:
OK, will do latter. Please open a ticket to track that change.